src/realm.h

```diff
@@ -1241,6 +1241,13 @@ RLM_API realm_schema_t* realm_get_schema(const realm_t*);
  */
 RLM_API uint64_t realm_get_schema_version(const realm_t* realm);
 
+/**
+ * Get the schema version for this realm at the path.
+ *
+ * This function cannot fail.
```

Comment:
This opens (and possibly creates) a Realm file, which can fail.

Reply:
Yep left over…